presto-main/src/test/java/com/facebook/presto/operator/TestExchangeClientConfig.java

```diff
@@ -55,6 +55,7 @@ public void testExplicitPropertyMappings()
                 .put("exchange.client-threads", "2")
                 .put("exchange.page-buffer-client.max-callback-threads", "16")
                 .put("exchange.acknowledge-pages", "false")
+                .put("exchange.initial-request-count", "42")
```

Comment:
nit: you probably need to update this test as well

Reply:
Updated